### PR TITLE
[HCAP-224] Refactor and transform participants data. 

### PR DESCRIPTION
**Pull request by fwpushan**
Update participates validation schema, processing and parsing.

### CODE REVIEW

**Comment by mauriciofreshw** [comment]
> Update participates validation schema, processing and parsing.

@fwpushan I removed the old lib and added a new one more optimized. The upload should work now.

**Review by mauriciofreshw — APPROVED** [review]

File: server/server.js [hunk 1]
```diff
@@ -116,8 +116,12 @@ app.post(`${apiBaseUrl}/employees`,
       return res.json({ status: 'Error', message: req.fileError });
     }
 
-    const response = await parseAndSaveParticipants(req.file);
-    return res.json(response);
+    try {
+      const response = await parseAndSaveParticipants(req.file);
+      return res.json(response);
+    } catch (excp) {
+      return res.status(400).send(`${excp}`);
```
Comment (ashtonmeuser) [inline server/server.js:123]: Just so we're all aware, we can throw inside the `asyncMiddleware`. The error will be presented as a generic 500 error in `error-handler.js`. If we see a whole bunch of these try/catch blocks, we might want to consider reworking the error handler.

**Review by ashtonmeuser — APPROVED** [review]

File: server/server.js (hunk 1 above)
Comment (fwpushan) [reply server/server.js:123]: OK, but I tried to throw error but it is not showing the appropriate error message.

Comment (fwpushan) [reply server/server.js:123]: I got it I will fix it different PR.

**Pull request merged**